lib/internal/test_runner/harness.js

```diff
@@ -142,7 +142,7 @@ function setup(root) {
     createProcessEventHandler('unhandledRejection', root);
   const coverage = configureCoverage(root, globalOptions);
   const exitHandler = async () => {
```

Comment:
```suggestion
  const exitHandler = () => {
```
and clear anwhere awaiting `exitHandler`